src/components/EditPartnership/EditPartnership.tsx

```diff
@@ -10,11 +24,89 @@ interface EditPartnershipProps {
 
 export const EditPartnership: FC<EditPartnershipProps> = ({
   state,
-  // partnership,
+  partnership,
 }) => {
+  const [updatePartnership] = useUpdatePartnershipMutation();
+  const [deletePartnership] = useDeletePartnershipMutation();
+  const { enqueueSnackbar } = useSnackbar();
+
+  const submit = async (input: UpdatePartnershipInput) => {
+    const inputWithId: UpdatePartnershipInput = {
+      partnership: { ...input.partnership, id: partnership.id },
+    };
+    await updatePartnership({
+      variables: { input: inputWithId },
+      refetchQueries: ['ProjectPartnerships'],
+    });
+    enqueueSnackbar(
+      `Updated Partnership with ${partnership.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
+  };
+
+  const onDeleteClick = async () => {
+    const { id } = partnership;
+    await deletePartnership({
+      variables: { input: id },
+      refetchQueries: ['ProjectPartnerships'],
+    });
+    state.onClose();
+    enqueueSnackbar(
+      `Deleted Partnership with ${partnership.organization.name.value}`,
+      {
+        variant: 'success',
+      }
+    );
+  };
+
+  // TODO: have DialogForm pass down initialValues to the radio options (currently passed to Checkboxes)
```

Comment:
I'll play around here and see if I can reproduce

Reply:
Passed through the initialValues, you'll see the checkboxes working but not the dialog